diffeqpy/de.py

```diff
@@ -1,6 +1,13 @@
 import os
 import sys
 
+from jill.install import install_julia
+from jill.install import last_julia_version
+
+if last_julia_version() == None:
```

Comment:
It's strange that CI doesn't output anything related to this.

`last_julia_version` might not be the function you want; it simply checks `~/.julia/environments/` folder and see what's the latest julia version people are using.

```suggestion
if shutil.which("julia") == None:
```